src_code/src/controllers/services/Inventory/GetAllModels.js

```diff
@@ -0,0 +1,65 @@
+/** Initialize neccessary modules */
+const responseBuilder = require("../../../utils/interfaces/IResponseBuilder");
+const db = require("../../../configurations/database/DatabaseConfigurations");
+
+/**
+ * Retrieves information about all equipment models and their respective equipment counts.
+ *
+ * @param {Object} res - Express response object for sending HTTP responses.
+ * @returns {Object} An HTTP response containing information about equipment models and their counts.
+ * If successful, returns a JSON response with status 200 and an array of model information.
```

Comment:
What model info?

Reply:
The information of the model, array of objects that we want.

Reply:
I just adjusted the comments a bit to make it clearer!